internal/common/expand_into_runfiles.bzl

```diff
@@ -15,63 +15,33 @@
 """Helper functions to expand paths into runfiles
 """
 
-def expand_location_into_runfiles(ctx, path):
-    """Expand a path into runfiles if it contains a $(location).
+def expand_location_into_runfiles(ctx, input, targets = []):
+    """Expands all $(location ...) templates in the given string by replacing $(location //x) with the path
+    in runfiles of the output file of target //x. Expansion only works for labels that point to direct dependencies
+    of this rule or that are explicitly listed in the optional argument targets.
 
-    If the path has a location expansion, expand it. Otherwise return as-is.
+$(location ...) will cause an error if the referenced target has multiple outputs. In this case, please use
+$(locations ...) since it produces a space-separated list of output paths. It can be safely used for a=
```

Comment:
nit: stray equals sign on the end

Reply:
not that it matters since I think we don't do docgen for this?

Reply:
these docs were updated